deps/repl.lua

```diff
@@ -62,14 +62,15 @@ setmetatable(exports, {
     for i = 1, results.n do
       results[i] = utils.dump(results[i])
     end
-    print(table.concat(results, '\t'))
+    stdout:write(table.concat(results, '\t'))
   end
 
   local buffer = ''
 
   local function evaluateLine(line)
+    stdout:write("\x1b[0G");
```

Comment:
In all the places print() is replaced, adding a newline is insufficient, because the cursor isn't reset to the left (so each time you evaluate a line, the output and subsequent prompt keep moving to the right).  

I ganked this command string from readline.lua, where it has the comment "Cursor to left edge".  I'll look for a cleaner way to do this, now that I know that you're interested in the change in principle.